src/test/feature/Ranger/test_ranger.cpp

```diff
@@ -24,19 +24,34 @@
 
 #include "lib/command.h"
 #include "lib/gpfdist.h"
-#include "lib/sql_util.h"
 #include "lib/string_util.h"
 
 using std::vector;
 using std::string;
 using hawq::test::SQLUtility;
 using hawq::test::Command;
 
+TestHawqRanger::TestHawqRanger()
+{
+	initfile = hawq::test::stringFormat("Ranger/sql/init_file");
+	rangerHost = getRangerHost();
+}
+
+std::string& TestHawqRanger::getRangerHost()
```

Comment:
I mean the python script "rangeruser.py" need ranger server host, but you gave it rps host.